test.py

```diff
@@ -9,8 +11,17 @@
 
 print oc.toRadString()
 
+
+genday = Gendaymtx(v=True,h=True)
+genday.inputFiles =[r'C:\Ladybug\annual\annualSimulation\State_College__Penn_State___Su_PA_USA.wea'] #This path needs to be present physically !
```

Comment:
@mostaphaRoudsari Okay. I still need to get my head around the commandBase class. I will keep this in my mind.